scripts/simex

```diff
@@ -912,7 +945,14 @@ invoke_parser.add_argument('--repetition', type=int) # Legacy argument for SGE.
 invoke_parser.add_argument('specfile', type=str)
 
 def do_internal_queuesock(args):
-	simexpal.queuesock.run_queue(args.sockfd, args.force)
+	try:
+		simexpal.queuesock.run_queue(args.sockfd, args.force)
+	except OSError as e:
+		if e.errno == errno.EADDRINUSE:
+			print("There is currently a queue daemon running, which possibly was not closed properly. "
+				  "Use 'simex queue daemon --force' to forcefully open a new daemon.")
```

Comment:
"was not closed" -> "did not terminate". "open" -> "launch". See do_queue_interactive.